deployment/hasura/metadata/actions.yaml

```diff
@@ -15,6 +15,10 @@ actions:
     definition:
       kind: synchronous
       handler: http://aerie_commanding:3000/put-expansion-set
+  - name: expandAllActivities
+    definition:
+      kind: synchronous
```

Comment:
What does this `synchronous` do?

Reply:
From https://hasura.io/docs/latest/graphql/core/actions/index/#kind
>Synchronous: Sync actions return a response to the client after receiving a response from the handler.

From memory, each of our `mutation`s need this.